prow/pjutil/pjutil.go

```diff
@@ -49,7 +49,7 @@ func NewProwJob(spec prowapi.ProwJobSpec, extraLabels, extraAnnotations map[stri
 			Kind:       "ProwJob",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        uuid.NewV1().String(),
+			Name:        uuid.New().String(),
```

Comment:
to check with the owners if is ok to change the version, this New() seems to generate a version 4.

For version 1 the generator seems to be NewUUID() https://github.com/google/uuid/blob/44b5fee7c49cf3bcdf723f106b36d56ef13ccc88/version1.go#L19

Reply:
Good catch, I thought I checked it did V1, but probably lost my thought somewhere along the way. Fixed.